prql-compiler/tests/integration/snapshots/integration__tests__rdbms@switch.prql.snap

```diff
@@ -3,14 +3,14 @@ source: prql-compiler/tests/integration/main.rs
 expression: result_string
 input_file: prql-compiler/tests/integration/queries/switch.prql
 ---
-Samuel Rosa,1071
-no composer,4884
-no composer,6373
-no composer,6635
-L. Muggerud,7941
-no composer,11650
-L. Muggerud,21211
-unknown composer,29048
-Gilberto Gil,32287
-Chico Science,33149
+Samuel Rosa
```

Comment:
OK nice, this was wrong before! From https://github.com/prql/prql/blob/33fb9e02348158b1c0d6543514771a81285c09e0/prql-compiler/tests/integration/queries/switch.prql

Reply:
Yeah, I didn't even notice :D